redesign/css/dwyl.css

```diff
@@ -60,6 +60,14 @@ body {
   font-size: 4rem;
 }
 
+.bw-s {
+  border-width: thin;
```

Comment:
Thanks for the observation @nelsonic, I created the custom class as `bw1` on tachyons was too thick. However I hadn't realised that `ba` had a finer width than `bw1`. I hadn't come across applying it because input fields automatically have borders. I'll refactor this 😄  🎉